tests/test_models/test_head.py

```diff
@@ -6,7 +6,7 @@
 
 
 class ExampleHead(BaseHead):
-    # use a ExampleHead to success BaseHead
+    # use a ExampleHead to succeed BaseHead
```

Comment:
What does `succeed BaseHead` mean?

Reply:
I am not sure, I believe the former author means something like 'inherit'. So I change it to the verb form.

Reply:
succeed -> test